app/models/host/managed.rb

```diff
@@ -786,6 +786,14 @@ def power
     end
   end
 
+  def supports_power_and_running?
```

Comment:
@domcleal : I added two tests to test/unit/host_test.rb. What do you think of them?